copulas/univariate/gaussian_kde.py

```diff
@@ -17,6 +17,25 @@ class GaussianKDE(ScipyWrapper):
     probability_density = 'evaluate'
     sample = 'resample'
 
+    def __init__(self, sample_size=None, *args, **kwargs):
+        self.sample_size = sample_size
+        super().__init__(*args, **kwargs)
+
+    def fit(self, X, *args, **kwargs):
+        self.constant_value = self._get_constant_value(X)
+
+        if self.constant_value is None:
+            if self.sample_size is not None:
+                model = scipy.stats.gaussian_kde(X)
```

Comment:
I would do `super.fit` instead of `scipy.stats.gaussian_kde`. Under the hoods is the same, but it is clearer that we are actually doing the same thing twice instead of doing two different things.

I would make the final implementation look like this:

```python
super().fit(X, *args, **kwargs)

if self.sample_size:
    X = self.sample(self.sample_size)
    super().fit(X, *args, **kwargs)
```